### ISSUE TITLE
explore nco solution to extracting and caching time series fields

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#225]
Rather than using the current `xarray` approach for extracting a time series of a single (or a few) variables, we should explore an nco-based approach.  Potentially important would be that the NCO approach could update the time-series cache file as new data becomes available without overwriting data that is already present.

**Comment by xylar** [comment]
`ncrcat` seems to be the command we want.  A typical command would be something like:
```
ncrcat -O -v xtime_startMonthly,xtime_endMonthly,timeMonthly_avg_activeTracers_temperature mpaso.hist.am.timeSeriesStatsMonthly.*.nc temp.nc
```

**Comment by czender** [comment]
Just noticed this issue today. You might find the ncrcat --rec_apn [option](http://nco.sf.net/nco.html#rec_apn) applies here. It augments existing record-based timeseries rather than writing them from scratch, and thus has significantly less I/O overhead.

**Comment by xylar** [comment]
Thanks, @czender.  Yep, that's what I'm using in the branch I've developed to address this issue.
https://github.com/xylar/MPAS-Analysis/blob/switch_to_ncrcat/mpas_analysis/shared/time_series/mpas_time_series_task.py#L267
The branch is just waiting for its turn to get merged...

**Comment by xylar** [comment]
Should be addressed by #271

**Comment by milenaveneziani** [comment]
this can be closed, right?
